.github/workflows/main.yaml

```diff
@@ -3,33 +3,31 @@ on: push
 
 jobs:
   linux:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
     steps:
     - name: Checkout code
       uses: actions/checkout@v1
-    - name: Test scratch
-      shell: bash --noprofile --norc -x -eo pipefail {0}
+    - name: Build images
       run: |
-        cd 2.1.0/scratch
-        docker build --tag nats:2.1.0-scratch .
-        run_id=$(docker run --detach nats:2.1.0-scratch)
-        sleep 1
-        test -n "$(docker ps --filter "id=${run_id}" --filter "status=running" --quiet)"
-        docker kill "$run_id"
-  windows:
-    runs-on: windows-latest
+        ./scripts/build-images.sh
```

Comment:
From what I understand this is building the image just for test. Could we call this `./scripts/test-build-images.sh`.